tests/python/relax/frontend/test_onnx_frontend.py

```diff
@@ -0,0 +1,1378 @@
+    check_correctness(model)
+
+
+def test_conv():
+    conv_node = helper.make_node("Conv", ["x", "w", "b"], ["y"])
+    nchw_shape = [3, 12, 32, 32]
+    graph = helper.make_graph(
+        [conv_node],
+        "conv_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, nchw_shape),
+            helper.make_tensor_value_info("w", TensorProto.FLOAT, [4, 12, 3, 3]),
+            helper.make_tensor_value_info("b", TensorProto.FLOAT, [4]),
+        ],
+        outputs=[helper.make_tensor_value_info("y", TensorProto.FLOAT, [3, 4, 30, 30])],
+    )
+
+    model = helper.make_model(graph, producer_name="conv_test")
+    check_correctness(model)
+
+
+def test_pow():
+    pow_node = helper.make_node("Pow", ["x", "y"], ["z"])
+    shape = [32, 32]
+    graph = helper.make_graph(
+        [pow_node],
+        "pow_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, shape),
+            helper.make_tensor_value_info("y", TensorProto.FLOAT, shape),
+        ],
+        outputs=[helper.make_tensor_value_info("z", TensorProto.FLOAT, shape)],
+    )
+
+    model = helper.make_model(graph, producer_name="pow_test")
+    check_correctness(model)
+
+
+def test_erf():
+    erf_node = helper.make_node("Erf", ["x"], ["y"])
+    shape = [32, 32]
+    graph = helper.make_graph(
+        [erf_node],
+        "erf_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, shape),
+        ],
+        outputs=[helper.make_tensor_value_info("y", TensorProto.FLOAT, shape)],
+    )
+
+    model = helper.make_model(graph, producer_name="erf_test")
+    check_correctness(model)
+
+
+def test_cumsum():
+    cumsum_node = helper.make_node("CumSum", ["x", "axis"], ["y"])
+    shape = [32, 32]
+    type_proto = onnx.TypeProto()
+    tensor_type_proto = type_proto.tensor_type
+    tensor_type_proto.elem_type = TensorProto.INT64
+    graph = helper.make_graph(
+        [cumsum_node],
+        "cumsum_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, shape),
+        ],
+        initializer=[helper.make_tensor("axis", TensorProto.INT64, (), [1])],
+        outputs=[helper.make_tensor_value_info("y", TensorProto.FLOAT, shape)],
+    )
+
+    model = helper.make_model(graph, producer_name="cumsum_test")
+    check_correctness(model)
+
+
+def test_squeeze():
+    squeeze_node = helper.make_node("Squeeze", ["x", "axis"], ["y"])
+    shape = [1, 32, 1, 32]
+    graph = helper.make_graph(
+        [squeeze_node],
+        "squeeze_test",
+        inputs=[
+            helper.make_tensor_value_info("x", TensorProto.FLOAT, shape),
+        ],
+        initializer=[helper.make_tensor("axis", TensorProto.INT64, [2], [0, 2])],
+        outputs=[helper.make_tensor_value_info("y", TensorProto.FLOAT, [32, 32])],
+    )
+
+    model = helper.make_model(graph, producer_name="squeeze_test")
+    check_correctness(model)
+
+
+def test_const():
```

Comment:
Perhaps it would be good to have tests for multiple types or for the edge case discussed in the implementation (not supporting strings)